jbehave-support-core/src/main/java/org/jbehavesupport/core/web/WebScreenshotEvent.java

```diff
@@ -0,0 +1,17 @@
+package org.jbehavesupport.core.web;
+
+import static org.jbehavesupport.core.web.WebScreenshotTypes.*;
+import org.springframework.context.ApplicationEvent;
+
+public class WebScreenshotEvent extends ApplicationEvent {
+    private Type type;
+
+    public WebScreenshotEvent(Object source, Type type) {
+        super(source);
```

Comment:
Just a question - do we use the source at all? If not does the user need to input it and cant we just supply some dummy value? Or is it needed due to some Spring Internal stuff?

Reply:
> public ApplicationEvent(Object source)
> Create a new ApplicationEvent.
> Parameters:
> source - the object on which the event initially occurred (never null)

I don't think we can send dummy value there :(